### PR TITLE
Bring Ruby 3.0.3 and Node 16.13.2 to the container matrix

### PR DESCRIPTION
**Pull request by atatarenko**
This PR brings the latest path version Ruby of 3.0 so that we can run a gradual update to 3.1 at a project where the devcontainer has been using.

**Question:** Is this change enough to expect the devcontainer of ruby-3.0.3 published?

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       matrix:
         node: ["16.0.0", "14.16.1", "12.22.1", "10.24.1"]
-        ruby: ["3.0.1", "2.7.3", "2.6.7", "2.5.9"]
+        ruby: ["3.0.3", "3.0.1", "2.7.3", "2.6.7", "2.5.9"]
```
Comment (texpert) [inline .github/workflows/build.yml:37]: Do you still need "3.0.1"?

Comment (atatarenko) [reply .github/workflows/build.yml:37]: @texpert yep. There might be another project that uses devcontainer besides mine current. They might still want to continue using 3.0.1 until they fix it in Gemfile or other Dockerfiles.

**Review by texpert — APPROVED** [review]
LGTM

**Comment by atatarenko** [comment]
UPD: Also added Node 16.13.2

**Comment by atatarenko** [comment]
Hey @dzhlobo –  are the changes of this PR enough to expect the devcontainer of ruby-3.0.3 & node-16.13.2 published?

**Comment by atatarenko** [comment]
Well, I'm going to merge this soon and if anyone has other considerations, please leave them in this PR and I'll incorporate them in the following PRs.

**Pull request merged**